synapse/cortex.py

```diff
@@ -375,6 +372,9 @@ def __init__(self, dirn):
         self.model = s_datamodel.Model()
         self.view = View(self, self.layers)
 
+        self.ontagadds = collections.defaultdict(list)
```

Comment:
This feels like we're replicating existing eventbus behavior

Reply:
A tiny bit, yes, but not all callbacks are best optimized as messages.  In this case, being able to have a function with specific arguments makes it easier on the module implementer etc. :)